db/sql/0_msar.sql

```diff
@@ -1301,7 +1328,9 @@ WITH attnum_cte AS (
         format('%L', col_def_obj ->> 'default')
     END,
     -- We don't allow setting the primary key column manually
-    false
+    false,
+    -- Set the description for the column
+    COALESCE(quote_literal(col_def_obj ->> 'description'), NULL)
```

Comment:
This is redundant, isn't it? `COALESCE` will only fall back to the next argument if the first turns out to be `NULL`. I think you can remove the `COALESCE` call entirely.

Reply:
Done in https://github.com/centerofci/mathesar/pull/3186/commits/90ed05cd0fe71ff6dd68acd254742158fca8edf6